Game.h

```diff
@@ -11,6 +11,9 @@
 #include <Looper.h>
 #include <SupportDefs.h>
 
+#define HIGHSCORE_FILENAME "Highscore.dat"
+#define HIGHSCORE_DIRECTORY "Haiku2048"
```

Comment:
There'll probably be another (more?) setting file added in the future: window size/position, board state etc.
Just call it "Haiku2048_DIRECTORY".